### DISCUSSION TITLE
Error in Processing ISCE Only 8 Steps

### DISCUSSION THREAD

**Discussion by Rofiatulumami** [alexisInSAR/EZ-InSAR#103] [Help]
Hi, excuse me. I would like to ask about the Processing of ISCE.  When i'm trying to run the ISCE processing only 8 steps appear, unlike the 13 steps in the manual book. Can this be said to be an error or is it like that? Please help, thank you. 
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/161690953/ef7a0a4d-f7ab-4364-ad80-74f2f52171b8)

**Comment by alexisInSAR** [comment]
Hi, 

This seems fine. 

Alexis
